src/data.py

```diff
@@ -58,7 +58,19 @@ def _readCSV(file):
         else:
             skiprows = 0
 
-        dt = pd.read_csv(file, encoding="iso-8859-1", skiprows=skiprows, delimiter=";")
+        # Alguns órgãos possuem delimitador e encoding específico
```

Comment:
Que tal uma documentação estilo docstring ? (o copylot ou IA em geral é muito bom em documentar funções)
Se puder, inclua uma descrição também no início de cada código: o que este script faz?